python/hail/keytable.py

```diff
@@ -786,12 +786,12 @@ def _typecheck(self):
     def write(self, output, overwrite=False):
         """Write as KT file.
 
-        ***Examples***
-
-        Write variant annotations of a dataset as KT file:
+        ***Example***
```

Comment:
Can you change this back to Examples? We were trying to be consistent across all methods.